docs/docs/code/nodejs/http-requests/README.md

```diff
@@ -490,87 +490,11 @@ export default defineComponent({
 
 [Copy this workflow](https://pipedream.com/@dylburger/stream-a-file-upload-p_6lC1d2Z/edit) to run this example.
 
-## Use an HTTP proxy to proxy requests through another host
```

Comment:
You should keep this example since people still need to make requests through their own proxies, so the Node.js code is really useful to have.